upgrading/manual_upgrades.adoc

```diff
@@ -286,12 +286,21 @@ IMAGE_VERSION=<tag>
 +
 Replace `<tag>` with `{latest-tag}` for the latest version.
 
-. In previous versions of {product-title}, master hosts were marked unschedulable
+. Modify host labels and parameters.
++
+In previous versions of {product-title}, master hosts were marked unschedulable
 by default by the installer, meaning that no pods could be placed on the hosts.
 Starting with {product-title} 3.9, however, the web console is removed from the
 master services and deployed as a pod on the masters, requiring that masters be
-marked schedulable. This also requires adding labels to certain hosts, which
-assigns _node roles_ (either `master` or `compute`).
+marked schedulable.
++
+During upgrade, if a value is not provided for the `openshift_schedulable`
+parameter, it is set to *true*. If a value is provided for
+`openshift_schedulable`, it is not changed.
```

Comment:
Since they're not running ansible during the manual upgrade I'd just omit this paragraph, you tell them to remove it from the inventory on 319 which accounts for any future ansible use. The `oc adm manage-node <hostname> --schedulable=true` will be the part that actually marks the node schedulable.
Otherwise lgtm